numberformat/proposed.emu

```diff
@@ -1138,18 +1138,18 @@
         The abstract operation PartitionNumberRangePattern creates the parts for a localized number range according to _x_ (which must be an Intl mathematical value), _y_ (which must be an Intl mathematical value), and the formatting options of _numberFormat_ (which must be an object initialized as NumberFormat). The following steps are taken:
       </p>
       <emu-alg>
-        1. If _x_ is *NaN* or _y_ is *NaN*, throw a *RangeError* exception.
+        1. If _x_ is ~not-a-number~ or _y_ is ~not-a-number~, throw a *RangeError* exception.
```

Comment:
Should this naming conventions be applied [here](https://github.com/tc39/proposal-intl-numberformat-v3/pull/82/files#diff-3b34a37f584f895af48c6bd33ec4cf03e92293eb17b96d0006cbebc6a81f3c57R1003)